taxi/templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+	<li> <a href="{%  url 'taxi:index' %}">Home page</a></li>
+	<li><a href="#">Manufacturers</a></li>
+	<li><a href="#">Cars</a></li>
```

Comment:
The link for 'Cars' is currently a placeholder. Please replace '#' with the correct URL template tag to ensure proper navigation.